### PR TITLE
Fix axes -> Axes changes in figure.py

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

Follow-up / correction of #18811.

### CODE REVIEW

File: lib/matplotlib/figure.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ class _AxesStack(cbook.Stack):
     Specialization of `.Stack`, to handle all tracking of `~.axes.Axes` in a
     `.Figure`.
 
-    This stack stores ``key, (ind, Axes)`` pairs, where:
+    This stack stores ``key, (ind, axes)`` pairs, where:
```
Comment (timhoffm) [inline lib/matplotlib/figure.py:59]: Usage in literals is similar to variables and thus should stay lowercase.

**Review by tacaswell — APPROVED** [review]
anyone can merge on CI green.

File: lib/matplotlib/figure.py (hunk 1 above)
Comment (lucyleeow) [reply lib/matplotlib/figure.py:59]: Whoops I forgot to mention this in the comments. I wasn't sure about this one. Thanks for fixing.

Comment (jklymak) [reply lib/matplotlib/figure.py:59]: I thought it was fine, but I'm so old I remember when everyone was excited about mixed-case variable names (an improvement from all-caps!)

**Pull request merged**